src/components/React/ControlsPanel.js

```diff
@@ -46,23 +45,23 @@ export default connect(null, {myChangeVolumeAction})
   render () {
     return (
       <div className="mx-1">
-        <Flexbox justifyContent="center" className="my-3 mx-3">
-          <Button 
-            icon="pi pi-arrow-right" 
-            label="Controls" 
+        <div className="d-flex justify-content-center m-3">
```

Comment:
What's the reasoning of moving away from react-bootstrap components?

Reply:
the ones removed weren't react-bootstrap, they were either `primereact` or `react-flexbox` - I'm trying to get us down to only using bootstrap components or straight bootstrap (for utilities).

Reply:
I am talking about the `<div className="d-flex justify-content-center m-3">` here

Reply:
The `<div className="d-flex justify-content-center m-3">` from here!

Reply:
I don't think there's a component for that.  It's just using utilities. Looking again...

Reply:
I don't see anything in `react-bootstrap` for the flex/padding utilities

Reply:
`d-flex` and `Container` are different, mostly unrelated things

Reply:
I thought the padding/alignment was controlled in the containers with Rows and Columns by controlling their size and alignments ... like in [here](https://react-bootstrap.github.io/layout/grid/).  It seemed to me like the initial `<Flexbox>` was just a container for the `Button` and `SelectButton`.  But I may be confused,  you more styling than I do and I'm not running the code just eyeing it!

Reply:
We probably *could* keep nesting rows and columns to do a similar thing (though pretty sure we'd still want the margin utility), but it would make the html a lot deeper as the `Row`/`Column` is row major and we just want our content to flow in a column here.  Is that worth it?

Reply:
I think it's only worth it if it helps with mobile behavior. If I recall, columns work out of the box for mobile...

Reply:
`d-flex` is what makes columns mobile friendly under the hood, so we've already got that 😄 .  (As an aside - I don't think this app is mobile usable - should we enforce that programmatically?)

Reply:
I think at least the general layout and menus/buttons and controls should be responsive 
![image](https://user-images.githubusercontent.com/1238840/134367723-a51fcc3f-1cd2-4162-88d0-d1b5a61bbdf4.png)

Reply:
But that could be another issue/PR

Reply:
I think the Controls button should stuck with the other ones at least so it remains readable

Reply:
#20

Reply:
Late to the conversation but the radio buttons make things a fair bit neater on mobile. I'm also thinking that app won't be beneficial on mobile